pyiron/atomistics/structure/atoms.py

```diff
@@ -1304,40 +1298,22 @@ def get_neighbors_by_distance(
                         (pbc are automatically taken into account)
             tolerance (int): tolerance (round decimal points) used for computing neighbor shells
             id_list (list): list of atoms the neighbors are to be looked for
-            boundary_width_factor (float): width of the layer to be added to account for pbc.
-            num_neighbors_estimate_buffer (float): Extra volume taken into account for the num_neighbors estimation
+            width_buffer (float): width of the layer to be added to account for pbc.
```

Comment:
Misses docs for `allow_ragged`.

Reply:
How do you manage to see all the details? Do you have a software?